crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -361,29 +367,14 @@ impl PgIndexerStore {
                         objects::df_name.eq(excluded(objects::df_name)),
                         objects::df_object_type.eq(excluded(objects::df_object_type)),
                         objects::df_object_id.eq(excluded(objects::df_object_id)),
-                    ),
-                    |excluded: StoredObject| (
```

Comment:
was this just extraneous?

Reply:
Yeah, it was only used for the MySQL integration, which has been removed. It was something that was missed when that was ripped out